github-actions/verify-pr-title/package.json

```diff
@@ -8,24 +8,37 @@
 		"build": "tsc",
 		"lint": "eslint src/**/*.ts",
 		"package": "npx ncc build",
-		"test": "mocha \"./test/**/*.test.ts\"",
+		"test": "jest",
 		"all": "npm run build && npm run lint && npm test && npm run package"
 	},
 	"dependencies": {
 		"@actions/core": "^1.10.1",
-		"@actions/github": "^6.0.0"
+		"@actions/github": "^6.0.0",
+		"jest": "^29.7.0",
+		"ts-jest": "^29.1.5"
+	},
+	"jest": {
```

Comment:
This is all copy-pasted from the outer `package.json`, sans the code coverage stuff.